tests/test_cli_tmux_compat_split_window_surface_ref.py

```
@@ -216,6 +233,77 @@ def assert_resplit_after_close(
+            )
+        finally:
+            server.shutdown()
+            server.server_close()
+            thread.join(timeout=2)
+
+        if proc.returncode != 0:
+            raise AssertionError(
+                f"{launcher_kind} split-window returned non-zero\n"
+                f"stdout={proc.stdout.strip()}\n"
+                f"stderr={proc.stderr.strip()}"
+            )
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)